inductiva/tasks/methods.py

```diff
@@ -0,0 +1,48 @@
+"""Methods to interact with the tasks submitted to the API."""
+import pathlib
+from typing import Optional, Type
+
+from absl import logging
+
+import inductiva
+from inductiva import api
+from inductiva.types import Path
+from inductiva.client import ApiClient
+from inductiva.client.apis.tags.tasks_api import TasksApi
+from inductiva.utils.data import (unpack_output)
+
+
+def get_task_status(task_id: str):
+    """Get the status of a simulation."""
+
+    api_config = api.validate_api_config(inductiva.api_key)
+
+    with ApiClient(api_config) as client:
+        api_instance = TasksApi(client)
+
+        status = api.get_task_status(api_instance, task_id)
+
+    return status
+
+
+def fetch_task_output(task_id: str,
+                      output_dir: Optional[Path] = None,
+                      return_type: Type = pathlib.Path) -> pathlib.Path:
```

Comment:
In the cases of the `if-else` block of lines 41-44, the return is not a path, but rather `None`. Adjust the type annotation.